engine/schema/src/main/resources/META-INF/db/schema-41310to41400-cleanup.sql

```diff
@@ -26,3 +26,6 @@ UPDATE `cloud`.`storage_pool` SET path="/var/lib/libvirt/images" WHERE path="/va
 
 -- remove (one of) duplicate unique indexes from Region table
 ALTER TABLE `region` DROP INDEX `id_3`;
+
+-- remove the old NetApp storage APIs (unsupported since 4.12) from role_permissions
+DELETE from `cloud`.`role_permissions` WHERE rule IN ('createPool', 'modifyPool', 'deletePool', 'listPools', 'associateLun', 'dissociateLun', 'createLunOnFiler', 'destroyLunOnFiler', 'listLunsOnFiler', 'createVolumeOnFiler', 'destroyVolumeOnFiler', 'listVolumesOnFiler');
```

Comment:
We want the DB changes in the 4.14 -> 4.15 upgrade path.

Reply:
> We want the DB changes in the 4.14 -> 4.15 upgrade path.

Moved to 4.15 upgrade path